### ISSUE TITLE
`fomu-dev` missing from /etc/hosts

### ISSUE THREAD

**Issue by xobs** [im-tomu/fomu-pi-gen#6]

**Comment by ewenmcneill** [comment]
Seems to be present now:

```
fomu@fomu-dev:/tmp $ grep fomu-dev /etc/hosts
127.0.1.1	fomu-dev
fomu@fomu-dev:/tmp $ 
```

Although that might be better as:

```
127.0.0.1 localhost
127.0.1.1 fomu-dev.local fomu-dev
```

or something like that.

Ewen

**Comment by xobs** [comment]
This is solved in v1.2.6.

**Comment by ewenmcneill** [comment]
@xobs, FYI,1.2.6 "just worked" for me -- unpacked, copied onto SD card, booted Pi, watched it first boot reboot a few times (partition resize), logged in, date was sane, `git clone` worked, and `make run` worked to get sensibly bright blinking LEDs (for bare board in daylight).  So I think we're converging on a useful initial Pi image.

Of note, on the date issue, it appears that the "Start Network Time Synchronisation" fails multiple times during the boot (possibly causing the issue you needed to work around?) because it's started well before the network interface is brought up ("Started Raise network interface" is quite late in the boot).  Possibly that "Network Time Synchronisation" target would benefit from a dependency on the "network online" target being achieved?  But at least it automatically works at this point just with warnings.

Ewen

```
fomu@fomu-dev:~/src/fomu-tests/blink $ cat /etc/hosts
127.0.0.1	localhost
::1		localhost ip6-localhost ip6-loopback
ff02::1		ip6-allnodes
ff02::2		ip6-allrouters

127.0.0.1	localhost
127.0.1.1	fomu-dev fomu-dev.local
fomu@fomu-dev:~/src/fomu-tests/blink $ 
```

(Yes there are two IPv4 `localhost` entries now, but they're the same, so that's safe.)
